AmplifyPlugins/API/Tests/APIHostApp/AWSAPIPluginGraphQLIAMTests/README.md

```diff
@@ -44,7 +44,7 @@ Do you want to use the default authentication and security configuration?
  Please enter a name for your identity pool. 
     <amplifyintegtestCIDP>
  Allow unauthenticated logins? (Provides scoped down permissions that you can control via AWS IAM) 
-    No
+    Yes
```

Comment:
what does this mean? why did we switch it to Yes?

Reply:
it is needed for `testunauthRole` tests